tasks/app/controllers/tasks_controller.rb

```diff
@@ -18,6 +18,28 @@ def create
     redirect_to "/"
   end
 
+  def edit
+    id = params[:id]
+    @task = Task.find(id)
+  end
+
+  def update
+    id = params[:id]
+    @task = Task.find(id)
+    @task.update(
```

Comment:
Because we have the `task_params` method defined using `params.permit` we can actually simplify this call to `@task.update(task_params[:task])`.